crates/matrix-sdk-crypto/src/machine/mod.rs

```diff
@@ -2816,6 +2939,28 @@ impl OlmMachine {
     }
 }
 
+fn sending_device_is_unverified(encryption_info: &EncryptionInfo) -> bool {
+    if let VerificationState::Unverified(level) = &encryption_info.verification_state {
```

Comment:
nit: I think that a full `match` with `Verified`/`Unverified` will be easier to read than the `if let` and `else`

Also the name `is_unverified` can be better? because now when `UnverifiedIdentity` is true then `is_unverified` is false.

Can we do something similar than for megolm? something around `check_sender_trust_requirement`?

Reply:
Thanks, improved with a `match` in 02ba63c832 and re-arranged to try and look more like `check_sender_trust_requirement` in 550fa11bb7 . What do you think?